bindings/java/c_kzg_4844_jni.c

```diff
@@ -242,14 +267,23 @@ JNIEXPORT jbyteArray JNICALL Java_ethereum_ckzg4844_CKZG4844JNI_computeBlobKzgPr
     return NULL;
   }
 
+  size_t commitment_bytes_size = (size_t)(*env)->GetArrayLength(env, commitment_bytes);
+  if (commitment_bytes_size != BYTES_PER_COMMITMENT)
+  {
+    throw_invalid_size_exception(env, "Invalid commitment size.", commitment_bytes_size, BYTES_PER_COMMITMENT);
+    return NULL;
+  }
+
   Blob *blob_native = (Blob *)(*env)->GetByteArrayElements(env, blob, NULL);
+  Bytes48 *commitment_native = (Bytes48 *)(*env)->GetByteArrayElements(env, commitment_bytes, NULL);
 
   jbyteArray proof = (*env)->NewByteArray(env, BYTES_PER_PROOF);
   KZGProof *proof_native = (KZGProof *)(uint8_t *)(*env)->GetByteArrayElements(env, proof, NULL);
 
-  C_KZG_RET ret = compute_blob_kzg_proof(proof_native, blob_native, settings);
+  C_KZG_RET ret = compute_blob_kzg_proof(proof_native, blob_native, commitment_native, settings);
 
   (*env)->ReleaseByteArrayElements(env, blob, (jbyte *)blob_native, JNI_ABORT);
+  (*env)->ReleaseByteArrayElements(env, commitment_bytes, (jbyte *)commitment_native, JNI_ABORT);
```

Comment:
Do we care that in some places ReleaseByteArrayElements does `mode == JNI_ABORT` and in others `mode == 0`?